chapter_13_database_layer_validation.asciidoc

```diff
@@ -173,6 +164,9 @@ so if we try to run `makemigrations`, it will report there's nothing to do:
 // CSANAD: maybe I'm missing something here but I don't think there would have
 //         been any changes detected here even with another DB. I don't remember
 // changing the model since the last migration.
+// Update: Checked and last time we touched models.py was in chapter 07 where we did
+// run makemigrations and then `migrate` in chapter 08. No changes to the model since.
+// Can I delete this part "There's a way of checking whether the constraint..."?
```

Comment:
fyi @hjwp

Reply:
i have some changes i want to tmake tot this chapter in general, maybe deliberately raising an actual IntegrityError.  ok to delete... or maybe comment out this section?  with a `////` ?